### PR TITLE
Adds configuration for MIM-H03 controller

### PR DESCRIPTION
**Pull request by kirichkov**
In this particular configuration a Heatpump is attached to a MIM-H03 Wi-Fi kit controller

### CODE REVIEW

**Comment by SebuZet** [comment]
Support for MIM-H03 Wi-Fi kit controller

File: mim-h03_heatpump.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  attributes:
+    current_temperature:
+      type: number
+      status_template: '{{ device_state.Temperatures.0.current }}'
+    current_outside_temperature:
+      type: number
+      status_template: '{{ device_state.Temperatures.2.current }}'
+    min_temp:
+      type: number
+      status_template: '16'
```
Comment (kirichkov) [inline mim-h03_heatpump.yaml]: This should become:

      status_template: '{{ device_state.Temperatures.0.minimum }}'

File: mim-h03_heatpump.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    max_temp:
+      type: number
+      status_template: '55'
```
Comment (kirichkov) [inline mim-h03_heatpump.yaml]: This should become:

      status_template: '{{ device_state.Temperatures.0.maximum }}'

**Comment by SebuZet** [comment]
Hi,
Should I click something more to make your changes available in my repo?
I've closed this PR and I'm not sure if should I do this.

**Pull request closed without merging**